### ISSUE TITLE
Add custom options from mongoose schema to json schema

### ISSUE THREAD

**Issue by vaggeliskls** [DScheglov/mongoose-schema-jsonschema#17]
I am trying to validate (**with custom ajv keywords**) the  uniqueness and existence of specific properties of a mongoose model.
It would be a nice feature to enrich the **__processOptions** function in **types.js** with options types: exists: true|false and unique: true|false or enable a optional input of a predefined array of custom options.
Example:  **required('mongoose-schema-jsonschema')(mongoose, ['unique', 'exists'] )**
Very nice work. Thank you.

**Comment by DScheglov** [comment]
Hi, @vaggeliskls 

I think about something like that:

```js
const config = required('mongoose-schema-jsonschema/config');

config({
  customFields: [
    'unique',
    'exists',
    { mongoose: 'some-field-from-mongoose', jsonSchema: 'some-field-to-jsonSchems; }
  ]
});
```

**Comment by DScheglov** [comment]
Implemented in the v1.4.0.
See: https://github.com/DScheglov/mongoose-schema-jsonschema#schema-build-configuration
